service/k8sclient/k8sclient.go

```diff
@@ -560,6 +548,45 @@ func (c *K8sClient) CreateXtraDBCluster(ctx context.Context, params *XtraDBParam
 	return c.kubeCtl.Apply(ctx, res)
 }
 
+func (c *K8sClient) generateXtraDBPasswords() (map[string][]byte, error) {
+	root, err := generatePassword(passwordLength)
```

Comment:
I would suggest to generate 6 passwords using a loop and then just use that array for assigning.